src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1673,4 +1682,22 @@ public String getName()
     {
         return name;
     }
+
+    public void addPushEvent(JSONObject event)
+    {
+      System.out.println("CONF ADDING PUSH EVENT");
+      try {
+        pushEventQueue.put(event);
```

Comment:
Limiting the number of events stored in the queue is probably one of the issues that we'll want to look into sooner rather than later; otherwise, we're opening ourselves to running out of memory. We could do a maximum number of events, timing events out, etc.